Section_08_Aditional_Examples.ipynb

```diff
@@ -575,32 +587,15 @@
     "<img src=\"./figures/Interpolation_example_linear.svg\" alt=\"Linear Interpolation\" style=\"height: 400px;\"/>"
    ]
   },
-  {
```

Comment:
Did you consider this example to be too bulky and messy?

Reply:
Yes. Having the additional quadratic method does not add anything useful other than a very backwards way of introducing a wrapper function which I don't think is critical for the students to know. I would rather move this to one of the weekly assignments.